release/ray_release/schema.json

```diff
@@ -154,7 +154,10 @@
 				},
 				"long_running": {
 					"type": "boolean"
-				}
+				},
+                "artifact_path": {
+                    "type": "string"
+                }
```

Comment:
lint please

Reply:
I don't know why github ui shows it like this, but in my editor, it looks like
<img width="380" alt="Screen Shot 2023-03-13 at 3 32 26 PM" src="https://user-images.githubusercontent.com/87673679/224847228-ee815f37-1505-4e47-9940-669c19b6d821.png">